settings.yaml

```
@@ -51,7 +51,7 @@ api:
   # Will create an admin account of first startup (password can be seen in the logs).
   # Set to true for increased security.
   authentication: true
-
+  jwtSecret: ""
```

Also add a newline above and below.

```suggestion
  # Secret key used for signing JWT tokens.
  # If empty, a random key will be generated on each startup (invalidating existing tokens).
  jwtSecret: ""
```